fcrypt/fcrypt_internal_test.go

```diff
@@ -33,12 +36,14 @@ import (
 	"os/exec"
 	"path"
 	"reflect"
+	"strings"
 	"testing"
 
 	"github.com/aoscloud/aos_common/aoserrors"
 	"github.com/aoscloud/aos_common/utils/cryptutils"
 	log "github.com/sirupsen/logrus"
 
+	"github.com/aoscloud/aos_common/utils/testtools"
```

Comment:
move aos_common/utils/testtools import to section where common already placed

Reply:
Done